cmd/update-main.go

```diff
@@ -169,6 +169,12 @@ func IsKubernetes() bool {
 	return os.Getenv("KUBERNETES_SERVICE_HOST") != ""
 }
 
+// IsPCF returns true if minio is running on Pivotal Cloud Foundry.
+func IsPCF() bool {
+	// Minio PCF start-up script should set MINIO_PCF.
+	return os.Getenv("MINIO_PCF") != ""
```

Comment:
Is this is the only way to know? isn't there something PCF specific so that we don't have to add?

Reply:
These are the env vars set:
```
MONIT_DATE=Thu, 26 Oct 2017 00:16:53 +0000
MONIT_HOST=localhost
PATH=/bin:/usr/bin:/sbin:/usr/sbin
PWD=/etc/sv/monit
MONIT_PROCESS_PID=0
MONIT_EVENT=Started
MONIT_PROCESS_MEMORY=0
SHLVL=1
MONIT_PROCESS_CPU_PERCENT=0
MONIT_SERVICE=minio-server
MONIT_PROCESS_CHILDREN=0
MONIT_DESCRIPTION=Started
_=/usr/bin/printenv
```

But not sure if we should rely on the `MONIT_*` vars as I could not find documentation on these vars. Safer to depend on a var of our own.

Reply:
https://mmonit.com/monit/ is actually a different project, looks like there is no other way.

Reply:
At least can we have a more MINIO_PIVOTAL_CLOUD_FOUNDRY - an expanded name to avoid any future confusion ?